src/index.js

```diff
@@ -31,12 +31,12 @@ function selectClue(game) {
     el = e.target;
     a = parseInt(el.dataset.if);
     b = parseInt(el.dataset.is);
-    if ( !isNaN(a) && !isNaN(b) && $(el).attr('data-done') !== 'done') {
+    if (!isNaN(a) && !isNaN(b) && $(el).attr('data-done') !== 'done') {
```

Comment:
These conditionals are getting tough to read through. If you have time, come back to this section of code to refactor the conditional logic.

Reply:
I agree they are not easy to read.